src/background.ts

```diff
@@ -37,6 +37,17 @@ const players: Player[] = [
     title: 'soundcloud',
     url: 'https://soundcloud.com/',
   },
+  {
+    controlQueries: {
+      next: '.next-button.ytmusic-player-bar',
+      play: '.play-pause-button.ytmusic-player-bar',
+      prev: '.previous-button.ytmusic-player-bar',
+    },
+    default: false,
+    tabQuery: '*://music.youtube.com/',
```

Comment:
I think here the URL should end with a wildcard, otherwise, the extension will only load when you are on the main page.
```suggestion
    tabQuery: '*://music.youtube.com/*',
```